packages/data-explorer/tests/e2e/specs/test.js

```diff
@@ -2,11 +2,70 @@
 // http://nightwatchjs.org/guide#usage
 
 module.exports = {
-  'default e2e tests': browser => {
+  'show table layout and card layout': browser => {
+    browser
+      .url(process.env.VUE_DEV_SERVER_URL)
+      .click('.jumbotron .btn.btn-primary.btn-lg')
+      .pause(300)
```

Comment:
could be no need for all the pauses , thats what the wait for element is for, suggest to use constant instead of int ( 30, 5000)